docs/DeveloperGuide.md

```diff
@@ -2,25 +2,36 @@
 layout: page
 title: Developer Guide
 ---
+
+## Introduction
+
+PeopleSoft is a desktop app for **calculating the salary for shift-based contractors**, optimized for use via a **Command Line Interface (CLI)** If you are a **HR manager** and you can type fast, PeopleSoft can get your payroll tasks done **much faster** than traditional GUI apps.
+
+**PeopleSoft helps to:**
+* Simplify the management of data
+* Reduce menial labour
+* Reduce mistakes due to human error in calculation / accidental edits
+* Helps employees be assured that their hours and pay are registered correctly in the system<br><br>
```

Comment:
instead of tossing line breaks here, would it be better if we just split out the ToC to a separate section? eg:

```
--------------------------------------------------------------------------------------------------------------------

## Table of Contents
{:toc}

--------------------------------------------------------------------------------------------------------------------
```